src/locales/nl/pages.yaml

```diff
@@ -748,61 +748,69 @@ addendum-50:
     Ter ere van de 50ste verjaardag van de vereniging is een lustrumboek gemaakt. Het addendum van dit lustrumboek is achter [deze link](/lustrum-50-addendum.pdf) te vinden.
 
 owee:
-  title: Krashna bij de OWee 2022
-  poster: /images/owee2022NL.png
+  title: Krashna bij de OWee 2023
+  poster: /images/owee2023.jpeg
   text: |
-    ## Zondag 21 augustus
-    ##### Schieparade (17:00-19:00)
-      Op zondagmiddag is Krashna te zien tijdens de Delftse Schie parade!
-      Deze is onderdeel van de aftrap van de Owee waarbij we met toeters en bellen voor iedereen zichtbaar zullen zijn.
-    ## Maandag 22 augustus
-    ##### Infomarkt (13:00-17:45)
-      Uiteraard zullen wij aanwezig zijn op de traditionele informatiemarkt, waar je al je vragen kunt stellen. Dus zoek ons kraampje op en kom gezellig langs!
-    ##### Krashna canon avond (20:00-22:00)
-      Maandagavond staat er een gezellige canonavond op de planning, waarbij er verschillende canonliederen gezongen zullen worden. Hou je nou van zingen of wil je kijken of het iets voor je is? Kom dan langs in Eetcafé de Tango waar je samen met enthousiaste Krashnezen kan komen zingen onder het genot van een drankje.
-    ## Dinsdag 23 augustus
-    ##### Yoga ochtend (9:30-10:30)
-      Heb je na een aantal vermoeiende dagen OWee zin om onder het genot van klassieke muziek wakker te worden? Dat kan! Samen met Krashna kan je op dinsdagochtend komen yogaën. De locatie voor deze activiteit zal via sociale media bekend worden gemaakt.
-    ## Woensdag 24 augustus
-    ##### Activiteitenmarkt en Maestro (11:00-15:00)
-      Tijdens de activiteitenmarkt op X kan je opnieuw langs komen om informatie te vragen over onze vereniging. Daarnaast zullen er verschillende ensembles optreden op het podium, en kan jij je dirigeer skills uitproberen bij een echte Krashna Maestro!
-    ##### Fiddle inn avond (21:00-23:00)
-      Op woensdag, de één na laatste avond van de Owee, organiseert Krashna wederom een activiteit in Cafe de Tango. Ditmaal is het thema volksmuziek. Heb je een instrument tot je beschikking of zin om te zingen? Kom dan langs en maak samen met ons muziek, wij zullen zorgen voor de benodigde bladmuziek. Uiteraard is er ook voldoende mogelijkheid om gezellig te kletsen!
+    We hebben veel zin om potentieel nieuwe leden van onze vereniging te ontmoeten in de OWee! In deze week zijn we natuurlijk aanwezig bij grote activiteiten zoals de Schieparade, de infomarkt en de activiteitenmarkt. Daarnaast organiseren we zelf verschillende (muzikale) activiteiten. Hier vind je ons schema voor de OWee en informatie bij de verschillende evenementen.  
+    
+    Wil je bij Krashna? Ga naar [krashna.nl/interest](krashna.nl/interest) of kom naar de open repetities en kijk of het wat voor je is:
```

Comment:
Why is this not here in english? I like it, let's add it to the english page too